android/src/main/java/com/middlewarereactnative/MiddlewareReactNativeModule.java

```diff
@@ -80,9 +80,9 @@ public void initialize(ReadableMap configMap, Promise promise) {
       .setGlobalAttributes(attributesFromMap(globalAttributes))
       .setDeploymentEnvironment(deploymentEnvironment)
       .disableActivityLifecycleMonitoring()
-      .build(Objects.requireNonNull(getReactApplicationContext().getCurrentActivity()).getApplication());
+      .build((Application) getReactApplicationContext().getApplicationContext());
 
-    if("true".equals(sessionRecording)) {
+    if ("true".equals(sessionRecording)) {
```

Comment:
We can use Boolean.TRUE constant.

Reply:
From react native js side the configuration is getting passed as string so need to check in this way. However will take in follow up to see if we can pass boolean from react to android native.

Reply:
Or we can also convert this to string here itself.

Reply:
Done fixed :white_check_mark: